components/PrimarySourceSetsComponents/SingleSet/SourceSetSources/SourceSetSources.module.css

```
@@ -60,7 +59,7 @@
   position: relative;
   height: 180px;
 
-  @media (min-width: $smallRem) {
+  @media (min-width: 40em) {
   }
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)